docs/release-notes/release-notes-0.7.1.md

```diff
@@ -0,0 +1,117 @@
+Joinmarket-clientserver 0.7.1:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.7.1>
+
+This release requires Python 3.6+ - see "Upgrading" section in the 0.7.0 release notes for more information if needed.
+
+This implementation includes support for **receiving BIP78 (payjoin) payments** (0.7.0 introduced sending them), in Qt GUI or on command line, using either current P2SH segwit wallets ('3' addresses) or native bech32 wallets.
```

Comment:
```suggestion
This implementation includes support for **receiving BIP78 (payjoin) payments** (0.7.0 introduced sending them), in Qt GUI or on command line, using either current P2SH segwit wallets ('3' addresses) or native bech32 wallets ('bc1' addresses).
```
Is there a reason why bech32 does not have an example?

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/701/commits/01aad72ef0a5413240fa8825ee75833cba994e11